test/services/smart-action-hook-service.test.js

```diff
@@ -26,7 +28,7 @@ describe('services > smart-action-hook', () => {
         .rejects.toThrow('hook must return an array');
     });
 
-    it('should return an array of fields', async () => {
+    it('should return an object containing an array of fields and a layout', async () => {
```

Comment:
can you add some layout elements in the test just to ensure the layout is correctly returned ?

Reply:
it is already tested here https://github.com/ForestAdmin/forest-express/blob/feat/support-layout-elements-in-smart-action-form-hooks/test/services/smart-action-form-layout-service.test.js#L170

Reply:
not really, If smart-action-hook-service is modified to return only empty arrays for the layout, no test will break. But yeah, maybe a bit useless to test, you can leave it like that